src/components/atoms/OcList/OcList.vue

```diff
@@ -14,5 +14,35 @@ export default {
   name: "OcList",
   status: "ready",
   released: "unreleased",
+  props: {
+    raw: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+  },
 }
 </script>
+
+<style lang="scss">
+.oc-list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  &-divider > :nth-child(n + 2) {
+    border-top: 1px solid var(--oc-color-border);
+    margin-top: var(--oc-space-small);
+    padding-top: var(--oc-space-small);
+  }
+}
+.oc-list-raw {
```

Comment:
Could you create the class one line above as `&-raw` for commonly used scss syntax?